Jenkinsfile

```diff
@@ -42,12 +42,14 @@ stage('Tests') {
         }
     }
     branches['jdk11'] = {
-        node('maven-11') {
-            timeout(60) {
-                checkout scm
-                sh 'mvn -B -ntp -Dset.changelist -Dmaven.test.failure.ignore clean install'
-                infra.prepareToPublishIncrementals()
-                junit 'target/surefire-reports/*.xml'
+        retry(count: 3, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
```

Comment:
`handleNonKubernetes: true` feels weird, shouldn't the condition be renamed?

Reply:
Well this is to handle the scenario on ci.jenkins.io where we are requesting a label which _currently_ is handled by the `kubernetes` plugin but admins have left open the possibility of switching to some other cloud if performance merits it.